services/QuillLMS/client/app/bundles/Staff/utils/comprehension/feedbackHistoryRatingAPIs.ts

```diff
@@ -9,3 +9,11 @@ export const createOrUpdateFeedbackHistoryRating = async (data) => {
   });
   return { error: handleApiError('Failed to edit feedback history, please try again.', response) };
 }
+
+export const massCreateOrUpdateFeedbackHistoryRating = async (data) => {
+  const response = await mainFetch('feedback_history_rating/mass_mark', {
+    method: 'PUT',
+    body: JSON.stringify(data)
+  });
+  return { error: handleApiError('Failed to edit feedback history, please try again.', response) };
```

Comment:
This message does not fully capture current behavior (as currently we might save some values, just not all of them).  Probably not a huge deal for an internal tool.